calyx-py/test/correctness/queues/strict_and_rr_queues/gen_strict_or_rr.py

```diff
@@ -157,7 +167,14 @@ def insert_queue(
                 ],
             ),
             len_decr,
-            restore_hot if not round_robin else ast.Empty            
+            (
+                pifo.reg_store(hot, og_hot.out)
```

Comment:
This is how I avoided the unused group error in case of RR. I add the group into the component `pifo` only if the Python-level boolean `round_robin` is false!